src/fetch-wrapper.ts

```diff
@@ -16,6 +16,10 @@ export default function fetchWrapper(requestOptions: RequestOptions) {
     requestOptions.request && requestOptions.request.log
       ? requestOptions.request.log
       : console;
+  const parseResponse =
+    requestOptions.request && typeof requestOptions.request.parseResponse === "boolean"
+      ? requestOptions.request.parseResponse
+      : true
```

Comment:
we do require Node 18+ since v17, so you can simplify the above code by using `requestOptions.request?.parseResponse`.

I think I'd check explicitly for `false` though

```suggestion
  const parseResponse = requestOptions.request?.parseResponse !== false
```

Reply:
Why cant i commit your suggestion?

Reply:
No idea, sorry 🤷🏼 I also cannot commit it for some reasons. Seems like a github bug to me

Reply:
It's because the diff you commented on is marked as outdated